e2e/playwright/support/ui/pages/boards_view.ts

```diff
@@ -53,6 +60,32 @@ export default class BoardsViewPage {
         expect(await this.editableTitle.getAttribute('value')).toBe('');
         await expect(this.page.getByTitle('(Untitled Board)')).toBeVisible();
     }
+
+    async createDuplicateBoard(boardName: string) {
+        await this.page.hover(`div[title='${boardName}']`);
+        await this.page.locator(`//div[@title='${boardName}']/following-sibling::div`).click();
+        await this.duplicateBoardButton.click();
+        await this.page.waitForResponse(response => response.url().includes('duplicate?asTemplate=false') && response.status() === 200);
+        expect(await this.editableTitle.getAttribute('value')).toBe(`${boardName} copy`);
+    }
+
+    async shouldHaveBoardName(boardName: string) {
+        await this.editableTitle.isVisible();
+        expect(await this.editableTitle.getAttribute('value')).toBe(boardName);
+    }
+
+    async deleteBoard(boardName: string) {
+        await this.page.hover(`div[title='${boardName}']`);
+        await this.page.locator(`//div[@title='${boardName}']/following-sibling::div`).click();
+        await this.deleteBoardButton.click();
+        await this.deleteBoardConfirmationButton.click();
+        await this.page.waitForResponse(response => response.url().includes('plugins/boards/api/v2/boards') && response.status() === 200);
+        await this.waitforElementToNotExists(`div[title='${boardName}']`)
```

Comment:
re: `waitforElementToNotExists`, I would recommend not to introduce too much custom assertion when we could use the existing assertions from Playwright. use of `expect` with `toBeHidden` should work well in this case.

```suggestion
        await expect(sidebarBoard).toBeHidden();
```

Reply:
got it Saturn